gammapy/catalog/utils.py

```diff
@@ -295,3 +297,158 @@ def select_sky_box(table, lon_lim, lat_lim, frame):
     mask = lon_mask & lat_mask
 
     return table[mask]
+
+
+def get_source_by_name(source, catalog, id_column='Source_Name'):
+    """
+    Get source catalog entry by source name.
+
+    Parameters
+    ----------
+    source : string
+        Source name.
+    catalog : `~astropy.table.Table`
+        Catalog table.
+    id_column : str (default = 'Source_Name')
+        Column name of the source names.
+
+    Returns
+    -------
+    source : `~astropy.table.Row`
+        Entry for the given source.
+    """
+    index = np.where(catalog[id_column] == source)[0]
+    if len(index) == 0:
+        raise ValueError('Source {0} not found in catalog!'.format(source))
+    else:
+        return catalog[index[0]]
+
+
+def to_ds9_region(catalog, radius=None, color='green', glon='GLON', unc_glon=None,
+                  glat='GLAT', unc_glat=None, label=None, label_position='top',
+                  label_additional_info=None, label_distance=1.2, marker='circle',
+                  dashed=False, width=3):
+    """
+    Write ds9 region file from catalog.
+
+    Currently works only for galactic coordinates.
+
+
+    Parameters
+    ----------
+    catalog : `~astropy.table.Table`
+        Catalog with position and size information.
+    glon : str ('GLON')
+        Catalog column containing galactic longitude position.
+    glat : str ('GLAT')
+        Catalog column containing galactic latitude position.
+    unc_glon : str (None)
+        Catalog column containing galactic longitude position uncertainty.
+    unc_glat : str (None)
+        Catalog column containing galactic latitude position uncertainty.
+    radius : str (None)
+        Catalog column containing extension information.
+    color : str ('green')
+        Valid ds9 color.
+    label : str (None)
+        Catalog column to use for the label.
+    label_position: str ('top')
+        Position of the region label. Either 'top' or 'bottom'.
+    label_distance: float (1.2)
+        Distance of the label from the regions center. Given
+        in multiple of the region's radius.
+    label_additional_info: dict
+        Additional info to be shown in the region label.
+    dashed : bool (False)
+        Dashed region line.
+    width : int
+        Width of the region line.
+
+    Examples
+    --------
+    This example reads Greens catalog from `gammapy.datasets` and writes it to a
+    ds9 region file.
+
+        >>> from gammapy.datasets import load_catalog_green
```

Comment:
Does this need to be indented?
I think we usually don't do this.

Can you check in the html docs how it looks and if it's necessary?